web/libs/datamanager/src/components/Filters/Filters.jsx

```diff
@@ -70,20 +69,20 @@ export const Filters = injector(({ views, currentView, filters }) => {
         )}
       </Elem>
       <Elem name="actions">
-        <Button type="primary" size="small" onClick={() => currentView.createFilter()} icon={<IconPlus />}>
+        <Button
+          size="small"
+          look="string"
+          onClick={() => currentView.createFilter()}
+          leading={<IconPlus className="!h-3 !w-3" />}
+        >
           Add {filters.length ? "Another Filter" : "Filter"}
         </Button>
 
         {!sidebarEnabled ? (
           <Tooltip title="Pin to sidebar">
-            <Button
-              type="link"
-              size="small"
-              about="Pin to sidebar"
-              onClick={() => views.expandFilters()}
-              style={{ display: "inline-flex", alignItems: "center", padding: 0, width: "var(--button-height)" }}
-              icon={<Icon icon={IconChevronRight} size={18} />}
-            />
+            <Button look="string" type="link" size="small" about="Pin to sidebar" onClick={() => views.expandFilters()}>
```

Comment:
I believe `about` was a mistake in both old and new versions?
```suggestion
            <Button look="string" type="link" size="small" tooltip="Pin to sidebar" onClick={() => views.expandFilters()}>
```

Reply:
Also "Unpin sidebar" lost this field in `FilterSidebar`

Reply:
plus we also need `aria-label` here